test-status/api.js

```diff
@@ -30,7 +30,7 @@ function createNewCheckParams(pullRequestSnapshot, type, subType, status) {
     owner: pullRequestSnapshot.owner,
     repo: pullRequestSnapshot.repo,
     name: `ampproject/tests/${type} (${subType})`,
-    head_sha: pullRequestSnapshot.head_sha,
+    head_sha: pullRequestSnapshot.headSha,
```

Comment:
How come `head_sha` remains in snake case?

Reply:
This is for the GitHub API, which is not using the same style guide as ours. I thought about making some extra compatibility layer for a hot second and decided it's not worth the hassle 😅